db/seeds.rb

```diff
@@ -83,22 +83,20 @@
   80 => 'sal12_library',
   81 => 'tanner_lib_circ'
 }.each do |location_id, old_slug|
-  location = Location.find(location_id)
+  # I believe that looking up by an ID is a mistake, so we guard against it not being found.
+  location = Location.find_by(id: location_id)
+  next unless location
   FriendlyId::Slug.find_or_create_by slug: old_slug, sluggable: location
 end
 
 {
-  'falconer' => 'Biology Library (Falconer)',
-  'swain' => 'Chemistry & Chemical Engineering Library (Swain)',
   'classics-library' => 'Classics Library',
   'branner' => 'Earth Sciences Library & Map Collections (Branner)',
   'eal' => 'East Asia Library',
   'cubberley' => 'Education Library (Cubberley)',
   'englib' => 'Engineering Library (Terman)',
-  'hila' => 'Hoover Institution Archives',
-  'hoover' => 'Hoover Institution Library',
```

Comment:
aren't these technically different? I have no idea if they have different hours in practice, but afaik you can request items from them separately

Reply:
They are now combined in the production data.